react/Autosuggest/__snapshots__/Autosuggest.test.js.snap

```diff
@@ -36,7 +36,9 @@ exports[`Autosuggest should render with label 1`] = `
         role="combobox"
         type="text"
         value=""
-      />
+      >
+        
+      </input>
```

Comment:
Is this just the way `hydrate()` renders elements?

Reply:
No, this is a change to the renderer in Enzyme.